Packs/Trello/pack_metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Trello",
+    "description": "Trello is a card-based activity tracker.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "Adam Baumeister",
+    "url": "https://www.paloaltonetworks.com/cortex",
```

Comment:
I'm going to modify the pack metadata a bit (remove the url, add category, tags, usecases and so on - feel free to add them yourself if you wish)